### DISCUSSION TITLE
Payment Model

### DISCUSSION THREAD

**Discussion by BHTech2024** [ubicloud/ubicloud#2092] [General]
Hello everyone, I would like to find out if there is a user payment model available in this project as an open source. I saw on your demo page that there is an option to pay for the selected package, that's why I'm asking.
Greetings from Bosnia and Herzegovina

**Comment by duartemarco** [comment]
You mean payment model as in the user pays for the service?

**Reply by BHTech2024** [reply]
Yes!

**Comment by byucesoy** [comment]
Hi @BHTech2024,

For managed services, you can sign up from [our console](https://console.ubicloud.com/) and start using Ubicloud by entering your credit card details. We bill resource usage by minute, at the end of the month. The prices includes the underlying hardware cost, so you wouldn't pay anything extra for the underlying hardware.

If you host Ubicloud yourself, then the only cost would be the price you would pay for the hardware provider. Though, it would require significant expertise to host Ubicloud yourself, which is why we are offering it as managed service.
